docs/source/recipes/deploying-kubeflow-locally-for-dev.md

```diff
@@ -59,44 +56,33 @@ In this example, we will be performing the steps on a MacOS system
 `kubectl get all --all-namespaces` and verifying that the Docker Desktop pods are present and in `Running` state   
 ![Elyra](../images/docker-desktop-kubectl.png)  
   
-## Install KubeFlow
+## Install KubeFlow Pipelines 
 
-- Add the `kfctl` binary to your `PATH`. Downloading `kfctl` was part of the [Requirements](#Requirements) setup.
-```bash
-export PATH=$PATH:"<location of where you extracted kfctl>"
-```
-- Create a name for your Kubeflow deployment
-```bash
-export KF_NAME=<pick a name>
-```
-- Add a location to where you want to store your deployment files on your local system  
-NOTE: These are just configuration files about the deployment itself, not the running applications.
-```bash
-export BASE_DIR=<local place to store deployment files>
-export KF_DIR=${BASE_DIR}/${KF_NAME}
-```
-- Set the configuration file used to deploy Kubeflow, in this example we are using v1.0.2
-```bash
-export CONFIG_URI="https://raw.githubusercontent.com/kubeflow/manifests/v1.0-branch/kfdef/kfctl_k8s_istio.v1.0.2.yaml"
-```
 - Deploy Kubeflow  
+
+From the root directory of Elyra:
 ```bash
-mkdir -p ${KF_DIR}
-cd ${KF_DIR}
-kfctl apply -V -f ${CONFIG_URI}
+export PIPELINE_VERSION=0.5.1
+kubectl apply -k "github.com/kubeflow/pipelines/manifests/kustomize/cluster-scoped-resources?ref=$PIPELINE_VERSION"
+kubectl wait --for condition=established --timeout=60s crd/applications.app.k8s.io
+kubectl apply -k "etc/kubernetes/kubeflow/"
 ```
```

Comment:
considering not all users will have elyra cloned locally, should we make this a url?

Reply:
replaced with URL